src/ui/component/common/channel-tooltip.jsx

```diff
@@ -0,0 +1,65 @@
+// @flow
+import * as React from 'react';
+import ToolTip from 'react-portal-tooltip';
+import TruncatedText from 'component/common/truncated-text';
+import ChannelThumbnail from 'component/common/channelThumbnail';
+
+type TooltipProps = {
+  uri: string,
+  title: ?string,
+  active: ?boolean,
+  parent: ?HTMLElement,
+  claimId: ?string,
+  thumbnail: ?string,
+  claimName: ?string,
+  channelName: ?string,
+  description: ?string,
+};
+
+const ChannelTooltip = (props: TooltipProps) => {
+  const { uri, title, active, parent, claimId, thumbnail, claimName, channelName, description } = props;
+
+  const bgColor = '#32373b';
```

Comment:
Can't access to the sass var from here :stuck_out_tongue: 
Should I add a `css-var` ?

Reply:
Never mind looks like there is one already :+1: